### PR TITLE
Add IT Films

### PR DESCRIPTION
**Pull request by greybax**
**[https://github.com/greybax/IT-films]**

**[IT-films is repo with all kind of movies related with developers, computers, hackers and etc.. I think it should have a place in awesome list because its all about developers and good movies]**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- I have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by wolfika** [comment]
Thank you for the contribution! Please make sure to read the instructions for creating a new awesome-list thoroughly. Your list seems to miss some fundamental points, like the naming convention and formal requirements. In order to have your PR merged, please read and comply with the **Requirements for your pull request** and **Requirements for your Awesome list** sections in your pull request comment.

**Comment by greybax** [comment]
@wolfika Thanks for your review! About naming convention as I understand `awesome-` prefix is not required (f.e https://github.com/enjalot/algovis#readme, https://github.com/RomanTsegelskyi/rbooks#readme, https://github.com/mhinz/vim-galore#readme and many more repos haven't this prefix but are presented in this awesome repo..)

Could you please specify in more details which requirements I should to follow?

**Comment by greybax** [comment]
@sindresorhus could you please take a look and add your 5 cents here about `awesome-` prefix?

**Comment by wolfika** [comment]
@greybax The requirements which you claim you complied with, but in fact did not are as follows:

- The repo should have `awesome-list` & `awesome` as GitHub topics.
- Has a Table of Contents section.
  - Should be named `Contents`, not `Table of Contents`.
  - Should be the first section in the list.
  - Should only have one level of nested lists, preferably none.
- Has an appropriate license.
  - That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**

As for the name, [`create-list.md`](https://github.com/sindresorhus/awesome/blob/master/create-list.md) says the list should be named `awesome-{list-name}`.

**Comment by sindresorhus** [comment]
@greybax ping

**Comment by greybax** [comment]
@sindresorhus will take care about this PR. Tomorrow will send updated version

**Comment by greybax** [comment]
- [x] * The repo should have `awesome-list` & `awesome` as GitHub topics.
- [x] * Has a Table of Contents section.

  - [x] * Should be named `Contents`, not `Table of Contents`.
  - [x] * Should be the first section in the list.
  - [x] * Should only have one level of nested lists, preferably none.
- [x] * Has an appropriate license.

  - [x] * That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**

- [x]  As for the name, [`create-list.md`](https://github.com/sindresorhus/awesome/blob/master/create-list.md) says the list should be named `awesome-{list-name}`.

Fixed all issues with [awesome-it-films](https://github.com/greybax/awesome-IT-films) repo which were mentioned by @wolfika , please check and if all is good I'd be very appreciate if this repo will be part of awesome list community :)

**Comment by greybax** [comment]
Added CCO version 1.0 license 
https://github.com/greybax/awesome-IT-films/blob/master/LICENSE

**Comment by greybax** [comment]
@wolfika @sindresorhus please take a look on PR?

**Comment by sindresorhus** [comment]
> * This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.

---

> * Be added at the bottom of the appropriate category.


---

> Includes a project logo/illustration whenever possible.
> \* **The image should be high-DPI. Set it to maximum half the width of the original image.**


---

Please remove the `github stars  github forks  GitHub followers` buttons in the readme. They are moot as they're available in the UI already.

---

The `GitHub issues` `GitHub pull requests` badges should be placed on the same line as the other badges.

**Comment by sindresorhus** [comment]
It would be nice if some of the entries, at least the highest rated ones had commentary from you and users. I think this should be a contributing requirement. For inspiration: https://github.com/sindresorhus/awesome-scifi#ex-machina-2015-79

File: readme.md [hunk 1]
```diff
@@ -415,6 +415,7 @@
 - [Stock Resources](https://github.com/neutraltone/awesome-stock-resources#readme)
 - [GIF](https://github.com/davisonio/awesome-gif#readme) - Image format known for animated images.
 - [Music](https://github.com/ciconia/awesome-music#readme)
+- [Films](https://github.com/greybax/it-films#readme)
```
Comment (sindresorhus) [inline readme.md]: The title is `IT Films`

Comment (greybax) [reply readme.md]: fixed

**Comment by greybax** [comment]
> > * This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
> 
> > * Be added at the bottom of the appropriate category.
> 
> > Includes a project logo/illustration whenever possible.
> > * **The image should be high-DPI. Set it to maximum half the width of the original image.**
> 
> Please remove the `github stars github forks GitHub followers` buttons in the readme. They are moot as they're available in the UI already.
> 
> The `GitHub issues` `GitHub pull requests` badges should be placed on the same line as the other badges.

Fixed all these comments

**Comment by greybax** [comment]
> It would be nice if some of the entries, at least the highest rated ones had commentary from you and users. I think this should be a contributing requirement. For inspiration: https://github.com/sindresorhus/awesome-scifi#ex-machina-2015-79

What about current movies?

**Comment by sindresorhus** [comment]
> What about current movies?

What about them?

**Comment by greybax** [comment]
> > What about current movies?
> 
> What about them?

Updated [contribution guideline](https://github.com/greybax/awesome-IT-films/blob/master/CONTRIBUTING.md) in accordance with your proposal

**Comment by sindresorhus** [comment]
> What about them?

Add commentary for some of the movies you've watched. To get it started and as an example on how users should do it.

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome/pull/1434#discussion_r259225594 is not fixed. Please re-read my comment.

**Comment by sindresorhus** [comment]
There are also other things from my previous comments that are not fixed yet.

File: readme.md [hunk 2]
```diff
@@ -390,6 +390,7 @@
 - [Podcasts](https://github.com/guipdutra/awesome-geek-podcasts#readme)
 - [Email Newsletters](https://github.com/vredniy/awesome-newsletters#readme)
 - [IT Quotes](https://github.com/victorlaerte/awesome-it-quotes#readme)
+- [IT-Films](https://github.com/greybax/awesome-it-films#readme)
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should:
> * Include a short description about the project/theme of the list. **It should not describe the list itself.**

Comment (greybax) [reply readme.md]: Not sure what do you mean?

Description is 
>A curated list of awesome movies, documentaries and TV series about hackers, geeks and IT in common, sorted by IMDB rating. If you know some good films which do not contain in these lists just fork it and send a pull request.

And its already has on https://github.com/greybax/awesome-it-films#readme page.

**Comment by greybax** [comment]
> [#1434 (comment)](https://github.com/sindresorhus/awesome/pull/1434#discussion_r259225594) is not fixed. Please re-read my comment.

fixed `IT-Films` -> `IT Films`

**Comment by greybax** [comment]
> > What about them?
> 
> Add commentary for some of the movies you've watched. To get it started and as an example on how users should do it.

Added comments as example to some movies. See my commit https://github.com/greybax/awesome-IT-films/commit/e07aa3339ef7cf4c181fca4f8c6019a874ac6044

File: readme.md (hunk 2 above)
Comment (sindresorhus) [reply readme.md]: The description here could then be something like this:

> Movies, documentaries and TV series about hackers, geeks and information technology.

Comment (greybax) [reply readme.md]: added description (see changes https://github.com/sindresorhus/awesome/pull/1434/commits/3deac218a8d84a3d25154b9144098ef64f2484d7)

**Comment by greybax** [comment]
Added description

**Comment by greybax** [comment]
@sindresorhus please let me know if I have to change something for passing through guidelines

**Comment by sindresorhus** [comment]
> The image should be high-DPI. Set it to maximum half the width of the original image.

Still not fixed.

---

It's supposed to be an **Awesome** list. I really don't think movies/series with a rating below 5 is a good fit. 

---

> It would be nice if some of the entries, at least the highest rated ones had commentary from you and users.

Where can I see this?

---

It looks very messy when every item has an image with a different size. I think you should set a certain height on the images for consistency. Some of the covers are also way too large. 300px height would probably be good.

**Pull request closed without merging**